<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>GeometryFactory</author>
 <class>Bounding_volumes</class>
 <widget class="QMainWindow" name="Bounding_volumes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGAL Bounding Volumes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/CGAL.qrc">
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="margin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File Tools</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear"/>
   <addaction name="actionLoadPoints"/>
   <addaction name="actionSavePoints"/>
   <addaction name="actionRecenter"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Visualization Tools</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionShowConvexHull"/>
   <addaction name="actionShowMinCircle"/>
   <addaction name="separator"/>
   <addaction name="actionShowMinEllipse"/>
   <addaction name="actionShowMinRectangle"/>
   <addaction name="actionShowMinParallelogram"/>
   <addaction name="actionShowPCenter"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionInsertRandomPoints"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionInsertPoint"/>
    <addaction name="actionShowConvexHull"/>
    <addaction name="actionShowMinCircle"/>
    <addaction name="actionShowMinEllipse"/>
    <addaction name="actionShowMinRectangle"/>
    <addaction name="actionShowMinParallelogram"/>
    <addaction name="actionShowPCenter"/>
    <addaction name="separator"/>
    <addaction name="actionRecenter"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="actionLoadPoints"/>
    <addaction name="actionSavePoints"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutCGAL">
   <property name="text">
    <string>About &amp;CGAL</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInsertRandomPoints">
   <property name="text">
    <string>&amp;Insert random points</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionInsertPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/Input.qrc">
     <normaloff>:/cgal/Input/inputPoint.png</normaloff>:/cgal/Input/inputPoint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Insert Point</string>
   </property>
   <property name="toolTip">
    <string>Insert Point</string>
   </property>
   <property name="statusTip">
    <string>Left: Insert point</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../icons/File.qrc">
     <normaloff>:/cgal/fileToolbar/fileNew.png</normaloff>:/cgal/fileToolbar/fileNew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionShowMinCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/Actions/icons/min_circle.png</normaloff>:/cgal/Actions/icons/min_circle.png</iconset>
   </property>
   <property name="text">
    <string>Minimum Enclosing &amp;Circle</string>
   </property>
   <property name="iconText">
    <string>Minimum Enclosing Circle</string>
   </property>
   <property name="toolTip">
    <string>Minimum Enclosing Circle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionLoadPoints">
   <property name="icon">
    <iconset resource="../icons/File.qrc">
     <normaloff>:/cgal/fileToolbar/fileOpen.png</normaloff>:/cgal/fileToolbar/fileOpen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Points...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSavePoints">
   <property name="icon">
    <iconset resource="../icons/File.qrc">
     <normaloff>:/cgal/fileToolbar/fileSave.png</normaloff>:/cgal/fileToolbar/fileSave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Points...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRecenter">
   <property name="icon">
    <iconset resource="../icons/Input.qrc">
     <normaloff>:/cgal/Input/zoom-best-fit</normaloff>:/cgal/Input/zoom-best-fit</iconset>
   </property>
   <property name="text">
    <string>Re&amp;center the viewport</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionShowConvexHull">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Bounding_volumes.qrc">
     <normaloff>:/cgal/Actions/icons/convex_hull.png</normaloff>:/cgal/Actions/icons/convex_hull.png</iconset>
   </property>
   <property name="text">
    <string>Convex Hull</string>
   </property>
  </action>
  <action name="actionShowMinEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Bounding_volumes.qrc">
     <normaloff>:/cgal/Actions/icons/min_ellipse.png</normaloff>:/cgal/Actions/icons/min_ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Minimum Enclosing Ellipse</string>
   </property>
  </action>
  <action name="actionShowPCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Bounding_volumes.qrc">
     <normaloff>:/cgal/Actions/icons/p_center.png</normaloff>:/cgal/Actions/icons/p_center.png</iconset>
   </property>
   <property name="text">
    <string>Minimum 3 Center</string>
   </property>
   <property name="toolTip">
    <string>Minimum 3 Center</string>
   </property>
  </action>
  <action name="actionShowMinRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Bounding_volumes.qrc">
     <normaloff>:/cgal/Actions/icons/min_quadrilateral.png</normaloff>:/cgal/Actions/icons/min_quadrilateral.png</iconset>
   </property>
   <property name="text">
    <string>Minimum Enclosing Rectangle</string>
   </property>
  </action>
  <action name="actionShowMinParallelogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Bounding_volumes.qrc">
     <normaloff>:/cgal/Actions/icons/min_parallelogram.png</normaloff>:/cgal/Actions/icons/min_parallelogram.png</iconset>
   </property>
   <property name="text">
    <string>Minimum Enclosing Parallelogram</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Bounding_volumes.qrc"/>
  <include location="../icons/File.qrc"/>
  <include location="../resources/CGAL.qrc"/>
  <include location="../icons/Input.qrc"/>
 </resources>
 <connections/>
</ui>
